tests/UbenchTest.php

```diff
@@ -96,4 +98,23 @@ public function testCallableWithArguments()
         $this->assertNotNull($bench->getMemoryUsage());
         $this->assertNotNull($bench->getMemoryPeak());
     }
+
+    public function testWasStart()
+    {
+        $bench = new Ubench();
+
+        $this->assertFalse($bench->wasStart());
```

Comment:
You forgot the tests :) but there is no CI,  I try to setup travis really quick :)

Reply:
omg, sorry, fixing ...